### PR TITLE
Concep entry php enums

### PR DESCRIPTION
**Pull request by RaffyRod**
### Type of Change

- Adding a new entry

### Checklist

-  [x]  All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/enums/enums.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  // functions and methods
+  function choose_starter(Starter $starter) { }
+}
+?>
+```
+
+By using the `Starter` Enum, now it is possible to enforce types when accepting or returning a values for starter:
+
+```
```
Comment (yangc95) [inline content/php/concepts/enums/enums.md:51]:
```suggestion
```shell
```

File: content/php/concepts/enums/enums.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+```php
+<?php
+enum Starter {
+  case Charmander;
+  case Squirtle;
+  case Bulbasaur;
+  }
+
+  // functions and methods
```
Comment (yangc95) [inline content/php/concepts/enums/enums.md:43]:
```suggestion
  // Functions and methods
```

File: content/php/concepts/enums/enums.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  function choose_starter(Starter $starter) { }
+}
+?>
+```
+
+By using the `Starter` Enum, now it is possible to enforce types when accepting or returning a values for starter:
```
Comment (yangc95) [inline content/php/concepts/enums/enums.md:49]:
```suggestion
By using the `Starter` enumeration, now it is possible to enforce types when accepting or returning a values for starter:
```

**Review by yangc95 — COMMENTED** [review]
@RaffyRod thanks for your patience! I conducted a review of this entry. If you have any questions or concerns please tag me

File: content/php/concepts/enums/enums.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Enumerations or Enums'
```
[the remark this answers is not in the document]
Comment (RaffyRod) [reply content/php/concepts/enums/enums.md]: @yangc95 changes added please check

File: content/php/concepts/enums/enums.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+Description: 'An Enumerations or Enum is an special or enumerated custom type that allows developers to have a fixed number of possible values.
```
[the remark this answers is not in the document]
Comment (RaffyRod) [reply content/php/concepts/enums/enums.md]: @yangc95 changes added please check

File: content/php/concepts/enums/enums.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
```
[the remark this answers is not in the document]
Comment (RaffyRod) [reply content/php/concepts/enums/enums.md]: @yangc95 changes added please check

**Comment by RaffyRod** [comment]
@yangc95 changes done, please check

**Comment by RaffyRod** [comment]
@yangc95 this is ready for review

**Review by yangc95 — APPROVED** [review]
@RaffyRod This is a great start for enums in PHP! I made a change which will help set this up for future iterations.

Looks good for a second review

**Comment by RaffyRod** [comment]
@yangc95 Thanks a lot, do I have any other pending steps, or just need to wait for another review?

**Review by KTom101 — APPROVED** [review]
@RaffyRod I just had a couple of minor changes I applied directly here:[233e22a](https://github.com/Codecademy/docs/pull/1886/commits/233e22a823c8fef21a419b3dc0c5b40e8adbfcbc). Looks good to go 👍

**Comment by RaffyRod** [comment]
@KTom101 Thanks a lot

**Comment by KTom101** [comment]
@RaffyRod Live link: https://www.codecademy.com/resources/docs/php/enums

Looks like everything is good and you're listed correctly as a contributor 🎉 Thank you for this entry please check back regularly to see if there are any other unclaimed [issues](https://github.com/Codecademy/docs/issues?page=2&q=is%3Aissue+is%3Aopen) you want to work on! 

Screenshot 2023-04-22 at 2 46 48 PM

**Pull request merged**